src/modules/notes/notes.controller.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Controller,
+  Get,
+  UseGuards,
+  Post,
+  Body,
+  Query,
+  UseInterceptors,
+  UploadedFile,
+  ParseFilePipe,
+  MaxFileSizeValidator,
+  Res,
+  Param,
+} from '@nestjs/common';
+import {
+  ApiBody,
+  ApiConsumes,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+  ApiBearerAuth,
+} from '@nestjs/swagger';
+import { Query as ExpressQuery } from 'express-serve-static-core';
+import { Response } from 'express';
+
+import JwtAuthGuard from 'modules/auth/utils/jwt.guard';
+import { FileInterceptor } from '@nestjs/platform-express';
+import Note from './entity/note.entity';
+
+import NotesService from './notes.service';
+import CreateNoteDto from './dto/create-note.dto';
+import { fileStorage } from './storage';
+
+const MAX_FILE_SIZE_BYTES = 1024 * 1024 * 5; // eslint-disable-line no-magic-numbers
+@Controller('notes')
+@ApiTags('Notes')
+export default class NotesController {
+  constructor(private notesService: NotesService) {}
+
+  // get all notes
+
+  @ApiOperation({ summary: 'Getting All Notes' })
+  @ApiResponse({ status: 200, type: [Note] })
+  @Get('/all')
+  @UseGuards(JwtAuthGuard)
```

Comment:
I think you need setup guard to full controller. no?

Reply:
applied guards to full controller